src/anbox/cmds/session_manager.cpp

```diff
@@ -131,19 +131,21 @@ anbox::cmds::SessionManager::SessionManager()
                       use_system_dbus_));
 
   action([this](const cli::Command::Context &) {
-    auto trap = core::posix::trap_signals_for_process(
+    core::posix::SignalTrap* trap = core::posix::trap_signals_for_process(
```

Comment:
No need to remove the use of auto

Reply:
This format is changed because the cycling-reference of shared_ptr is checked in valgrind, I replace it with pointer with explicit new and delete. This time the valgrind doesn't complain it.

Reply:
I've revert this commit.